spec/views/index_posts_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Hi, Kindly note that your test failed.
- 
![Capture](https://github.com/Luiscarlosvd/blog-app-rails/assets/63972670/3815aa74-b7c5-4ba9-9b13-b7d46e524fd5)

Reply:
Hi! You need to run first the `rails tailwindcss:watch` comand to compile the tailwind and then run the tests. 😁